flask_multipass/providers/shibboleth.py

```diff
@@ -17,6 +17,13 @@
 
 def _lower_keys(iter_):
     for k, v in iter_:
+        if isinstance(v, str):
```

Comment:
i think that's always the case nowadays, or can you somehow get bytes there?

Reply:
Probably not bytes but I recall in other Flask projects some environ keys were being returned with type `int` (e.g. port numbers) or `float` (e.g. apache or wsgi version numbers) which I had to exclude from the encode/decode cycle, AFAIR.  
(This is not to say that that I know this conditional is required.)